### ISSUE TITLE
Server console.log error 

### ISSUE THREAD

**Issue by Dominis13** [williamledoux/meteor-static-server#1]
Hi. I'm trying to use this package and it's work fine but for each access request for file i see in my server console.log this error

```
W20150421-19:05:49.554(3)? (STDERR) Error: Meteor code must always run within a Fiber. Try wrapping callbacks that you pass to non-Meteor libraries with Meteor.bindEnvironment.
W20150421-19:05:49.554(3)? (STDERR)     at Object.Meteor._nodeCodeMustBeInFiber (packages/meteor/dynamics_nodejs.js:9:1)
W20150421-19:05:49.554(3)? (STDERR)     at [object Object]._.extend.get (packages/meteor/dynamics_nodejs.js:21:1)
W20150421-19:05:49.554(3)? (STDERR)     at [object Object].RouteController.lookupOption (packages/iron:router/lib/route_controller.js:66:1)
W20150421-19:05:49.554(3)? (STDERR)     at new Controller.extend.constructor (packages/iron:router/lib/route_controller.js:26:1)
W20150421-19:05:49.554(3)? (STDERR)     at [object Object].ctor (packages/iron:core/lib/iron_core.js:88:1)
W20150421-19:05:49.555(3)? (STDERR)     at Function.Router.createController (packages/iron:router/lib/router.js:201:1)
W20150421-19:05:49.555(3)? (STDERR)     at Function.Router.dispatch (packages/iron:router/lib/router_server.js:39:1)
W20150421-19:05:49.555(3)? (STDERR)     at Object.router (packages/iron:router/lib/router.js:15:1)
W20150421-19:05:49.555(3)? (STDERR)     at next (/home/dominis/.meteor/packages/webapp/.1.2.0.1n8nt51++os+web.browser+web.cordova/npm/node_modules/connect/lib/proto.js:190:15)
W20150421-19:05:49.555(3)? (STDERR)     at resume (/home/dominis/.meteor/packages/williamledoux_static-server/.0.1.2.xzsl9o++os+web.browser+web.cordova/npm/node_modules/connect/lib/middleware/static.js:60:7)
```

Files accessible, but errors very confuse.

**Comment by Dominis13** [comment]
Error found - it happens when requesting file that doesn't exists.
But i think better is add some log 'Requested file are missing' or totally nothing. Trace not good decision.

**Comment by williamledoux** [comment]
Thanks for your feedback, it should be fixed in v0.2.0
